### PR TITLE
feat: migration angular 15

### PR DESCRIPTION
**Pull request by QuCMGisaia**

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
 
-<div mat-dialog-content class="add_bookmark">
-  <h1 class="modal_title" mat-dialog-title>{{ 'Dataset name' | translate }} </h1>
-  <mat-form-field>
-    <input matInput tabindex="1"  [formControl]="bookmarkName" class="bookmark-search-input" (keypress)="pressEvent($event)">
-    <mat-error *ngIf="bookmarkName.invalid">{{errorMessage}}</mat-error>
+<div mat-dialog-content class="bookmark__add">
+  <h1 class="bookmark__add--title" mat-dialog-title>
+    {{ 'Dataset name' | translate }}
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html:4]: Normally we renamed all the places were it was 'Dataset' by Bookmar. I think this title was not changed precedently

File: projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.scss [hunk 2]
```diff
@@ -16,3 +16,49 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+// TODO: To override globally
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.scss:20]: there is a todo here

File: projects/arlas-toolkit/src/lib/components/language-switcher/language-switcher.component.scss [hunk 3]
```diff
@@ -74,11 +74,29 @@
...
+  &__trigger {
+    width: 24px;
+    height: 24px;
+    display: block;
+  }
 }
 
-::ng-deep .mat-select-trigger{
-  height: 24px;
+// TODO: remove overlap between checkbox and flag
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/components/language-switcher/language-switcher.component.scss:92]: trailing todo

File: projects/arlas-toolkit/src/lib/components/permissions-creator/permissions-creator.component.html [hunk 4]
```diff
@@ -1 +1 @@
-<button *ngIf="show" mat-stroked-button (click)="createPermission()" [matTooltip]="'Create permission from filters' | translate" class="permission">{{'Create permission' | translate}}</button>
+<button mat-stroked-button (click)="createPermission()" [matTooltip]="'Create permission from filters' | translate" class="permission">{{'Create permission' | translate}}</button>
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/components/permissions-creator/permissions-creator.component.html:1]: *ngIf="show" was removed

File: projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html (hunk 1 above)
Comment (QuCMGisaia) [reply projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html:4]: In the i18n file I still have all the keys with dataset and not bookmark. I think we just changed the translation

**Review by MohamedHamouGisaia — APPROVED** [review]

File: projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html:4]: ok !

**Review by sebbousquet — APPROVED** [review]

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**